File: LogViewer/LogViewerFrontend/src/app/Models/LogEntryResponse.ts
import { LogEntry } from './LogEntry';

export class LogEntryResponse {
    totalEntries: number;
    currentPage: number;
    logEntries: LogEntry[];

    static getEmptyResponse(): LogEntryResponse {
        return {
            totalEntries: 0,
            currentPage: 0,
            logEntries: []
        };
    }
}

File: LogViewer/ViewModels/LogEntryResponse.cs
using LogViewer.Core.Data.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LogViewer.ViewModels
{
    public class LogEntryResponse
    {
        public long TotalEntries { get; set; }
        public int CurrentPage { get; set; }
        public IEnumerable<LogEntry> LogEntries { get; set; }
    }
}
